synapse/lib/stormtypes.py

```diff
@@ -3451,7 +3469,8 @@ def getObjLocals(self):
     async def _methTeleOpen(self, url):
         url = await tostr(url)
         scheme = url.split('://')[0]
-        self.runt.confirm(('lib', 'telepath', 'open', scheme))
+        if not self.runt.allowed(('lib', 'telepath', 'open', scheme)): # pragma: no cover
```

Comment:
Can we add coverage for all of these permission checks?

Reply:
We probably could but it seems like a lot of redundant effort

Reply:
( i'll work on it later this evening )